airflow/providers/openlineage/conf.py

```diff
@@ -53,7 +58,11 @@ def disabled_operators() -> set[str]:
 
 @cache
 def selective_enable() -> bool:
-    return conf.getboolean(_CONFIG_SECTION, "selective_enable", fallback=False)
+    """[openlineage] selective_enable."""
+    option = conf.get(_CONFIG_SECTION, "selective_enable", fallback="")
+    if _is_true(option):
+        return True
+    return False
```

Comment:
```suggestion
    return _is_true(option)
```

Reply:
Should this just catch the exception from `getboolean` though?

Reply:
Thanks, those lines are a leftover after some logging statements, it should just return `_is_true` as you proposed. 

The `_is_true()` function was mainly introduced for the legacy env variables that are not part of Airflow config like `OPENLINEAGE_DISABLED` or `OPENLINEAGE_AIRFLOW_DISABLE_SOURCE_CODE` that need to be checked against true values. Then i simply used it everywhere for consistency. Yes, with Airflow configuration i could simply do getboolean + try/except instead of get + _is_true().

Maybe i could even skip the try/except. All bool variables that we use are False by default and user need to explicitly enable them, so my idea (maybe wrong?) was that i only look for values corresponding to true, but if it's not convertible i simply treat it as False, f.e. if somebody uses "yes" as the value of `OPENLINEAGE_DISABLED`. Maybe that's wrong and I should simply raise the error if the value is not a proper boolean, no matter if it's Airflow config or legacy env vars? I know that @JDarDagran had a different opinion on that than i do. Maybe let's agree on something here and stick to that, WDYT?

Reply:
I think it's crucial for us not to break Airflow or OpenLineage, so we should not raise any errors - when something is wrong with configuration (f.e. random string provided in place of boolean), let's just use default value which is False for our booleans. So i'd stay with the current approach - conf.get + _is_true().